Bottom: 560e2ba6b2dfc981ddb3992bd2db92d716bd524c
Top:    b879bb68f09a79f76a5b59c736bc3753c31f111e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 02:22:03 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/crosshair.c b/src/crosshair.c
index ac22277..aaa46ae 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -694,8 +694,10 @@ CrosshairOn (bool BlockToo)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -708,8 +710,10 @@ CrosshairOff (bool BlockToo)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -738,7 +742,7 @@ HideCrosshair (bool BlockToo)
   CrosshairStack[CrosshairStackLocation] = Crosshair.On;
   CrosshairStackLocation++;
 
-//  CrosshairOff (BlockToo);
+  CrosshairOff (BlockToo);
 }
 
 /* ---------------------------------------------------------------------------
@@ -758,11 +762,11 @@ RestoreCrosshair (bool BlockToo)
 
   if (CrosshairStack[CrosshairStackLocation])
     {
-//      CrosshairOn (BlockToo);
+      CrosshairOn (BlockToo);
     }
   else
     {
-//      CrosshairOff (BlockToo);
+      CrosshairOff (BlockToo);
     }
 }
